site/content/community/_index.html

```diff
@@ -0,0 +1,48 @@
+---
+title: "Community"
+layout: section
+---
+
+<div class="wrapper sub-page">
+    <h2>Do you want to help build Velero?</h2>
```

Comment:
@amymanion `h1`

Reply:
There's already a different h1 on the page.